asv_bench/benchmarks/io/parsers.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from pandas._libs.tslibs.parsing import _does_string_look_like_datetime
+
+from pandas.io.parsers import _concat_date_cols
+
+
+class DoesStringLookLikeDatetime(object):
+
+    params = (['2Q2005', '0.0', '10000'],)
+    param_names = ['value']
+
+    def setup(self, value):
+        self.objects = [value] * 1000000
+
+    def time_check_datetimes(self, value):
+        for obj in self.objects:
+            try:
+                _does_string_look_like_datetime(obj)
+            except ValueError:
```

Comment:
is this supposed to try/except?

Reply:
Yes, it is necessary to compare our version of the function with the old version, that raises `ValueError`

Reply:
can you add a TODO / comment here

Reply:
I was wrong.
try/except is not needed